scripts/badger/bridge_out_arbitrum_fees/remove_liq_sushiswap.py

```diff
@@ -1,41 +1,40 @@
-from rich.console import Console
-from brownie import interface, chain, Contract, accounts
+from brownie import interface, chain
 from helpers.addresses import registry
+from great_ape_safe import GreatApeSafe
 
-CONSOLE = Console()
 
 # slp tokens to liquidated
-slp_tokens = ["slpWbtcEth", "slpSushiWeth", "slpCrvWeth"]
+slp_tokens =[
+    registry.arbitrum.treasury_tokens.slpWbtcEth,
+    registry.arbitrum.treasury_tokens.slpSushiWeth,
+    registry.arbitrum.treasury_tokens.slpCrvWeth
+    ]
+
 
 DEADLINE = 60 * 60 * 12
 MAX_SLIPPAGE = 0.02
 
-ACCOUNT_TO_LOAD = ""
-
-
-def main(broadcast="true"):
-    dev = accounts.load(ACCOUNT_TO_LOAD)
 
-    safe = interface.IMultisigWalletWithDailyLimit(
-        registry.arbitrum.badger_wallets.dev_multisig_deprecated
-    )
+def main():
+    safe = GreatApeSafe(registry.arbitrum.badger_wallets.dev_multisig)
+    router = safe.contract(registry.arbitrum.sushi.router)
 
-    router = Contract(registry.arbitrum.sushi.router)
+    safe.take_snapshot(tokens=slp_tokens)
 
-    for key in slp_tokens:
-        slp = interface.IUniswapV2Pair(registry.arbitrum.treasury_tokens[key])
+    for address in slp_tokens:
+        slp = interface.IUniswapV2Pair(address, owner=safe.address)
 
         slp_balance = slp.balanceOf(safe)
```

Comment:
i think here may be worthy to add below a check if `slp_balance` is greater than zero to continue, similar in the `remove_liq_swapr` script